apptests/validator.py

```diff
@@ -0,0 +1,110 @@
+import argparse
+
+from youtube_transcript_api import YouTubeTranscriptApi
+
+import models
+
+
+class ModelClassValidator:
+    def __init__(self, class_name, language, model_type):
+        module_attrs = vars(models)
+        model_class = module_attrs.get(class_name)
+        self.model_instance = model_class()
+        self.language = language
+        self.model_type = model_type
```

Comment:
`__init__` should only accept objects that are instances of the `TranscriptTest` class. The creation of model_class should be in main function.